### ISSUE TITLE
Failed root on Goggles V2 after Butter push to gp150_01.00.0606_recovery

### ISSUE THREAD

**Issue by firstlily** [fpv-wtf/wtfos-configurator#327]
Ran Butter on Goggles V2 that had previously had fw V01.02.0015. Butter claimed success (command line output shown below) but WTFOS configurator fails to root.  Same issue on many different USB cables, including the one that I had performed Butter procedure.
Attempted reboot and re-ran butter (same success) and WTFOS root (same failure).  Please see enclosed screen-captures.

Using Windows 11-x64, Chrome Version 111.0.5563.147 (Official Build) (64-bit)
One oddity that I noticed is on the home WTFOS configurator screen, it does have an alert message saying "Could not connect to device, make sure that adb server is not running on your machine (adb kill-server) and that you are not connected in another tab/window." but this was immediately after reboot; I did not start up and Adb server, and none show in my task manager.  I chalked this up to the goggles not being rooted yet, but maybe there is a different reason?

Can you help?
![WTFOS screen 4](https://user-images.githubusercontent.com/25711209/228994104-0719200b-b57d-43e8-b1d2-33c03ca4638d.jpg)
![WTFOS screen 1](https://user-images.githubusercontent.com/25711209/228994108-8d12ad66-4622-476c-80e7-85a931d7c055.jpg)
![butter_output_screen](https://user-images.githubusercontent.com/25711209/228994109-2fb6d4f9-8f4d-4855-b95e-bb0b82554b33.jpg)
![WTFOS screen 3](https://user-images.githubusercontent.com/25711209/228994110-0ca1f62a-e114-4e36-96af-269a7243d605.jpg)

**Comment by stylesuxx** [comment]
Don't flash random recovery images (Who exactly suggested flashing that image?) They are there for a reason V2 Goggles you flash gp150 as described in the repository.

Step 3 failing might be a false positive. Try power-cycling your goggles and try again. Make sure your are in DIY mode.

**Comment by firstlily** [comment]
@stylesuxx , I got that from the readme on the main page, "if all else fails" to go to the recovery I mentioned.  That said, I re-ran butter just now, with the *correct* gp150_01.00.0606_recovery and then following with attempted re-root using the configurator page, but it gave me the same issues as above.  Power cycling did not give joy, and "DJI Digital FPV System" is what you mean by "DIY" then yes, I confirmed that it is and has been in that mode since running butter.  I even tried switching it to "DJI FPV" and back, but got the same results (after power cycling each time".  For reference, the text the configurator most recently gave is:
00:11 Attempting Step 1...

00:37 - Found Device: GP150, Version: 01.02.0015

00:37 Step 1 - Success! Rebooting...

00:56 Attempting Step 2...

01:02 Step 2 - Success!

01:05 Attempting Step 3...

01:07 Failed! Your firmware might not be rootable, please try downgrading with [Butter](https://github.com/fpv-wtf/butter).
 
The configurator pretty much confirms that the step-3 failure is stopping it from installing the WTFOS so seems like root does fail.

**Comment by stylesuxx** [comment]
No, the error it is showing at this point is simply wrong, please try power cycling when you see it and re-attempt. It should go through eventually. "DJI digital FPV system" is fine.

**Comment by firstlily** [comment]
Managed to get past all five steps.  For me the secret step seems to have been to remove the SD card from the goggles.
00:02 Attempting Step 1...

00:28 - Found Device: GP150, Version: 01.00.0607

00:28 Step 1 - Success! Rebooting...

00:49 Attempting Step 2...

00:54 Step 2 - Success!

00:57 Attempting Step 3...

00:59 Step 3 - Success!

00:59 Attempting Step 4...

01:06 Step 4 - Success! Rebooting...

01:29 Attempting Step 5...

01:29 All done - your device has been successfully rooted!

Still,  it is not able to install WTFOS after rooting, shows a message on the top of the screen saying:
"Could not connect to device, make sure that adb server is not running on your machine (adb kill-server) and that you are not connected in another tab/window."
After seeing "Connect your rooted Goggles or Air Unit via USB, make sure they are powered on and hit the connect button.", then powering them on, I only see the above message (about the ADB server) and then when pressing "CONNECT TO DEVICE", I see the choice for "pigeon - Paired" only, but it still does not connect if I choose that one; it just bounces back to the home screen.

I managed to fix the WTFOS install by forcing boot into safe mode (holding the goggle's binding button" for about 15 seconds while powering them on.

So it seems to be working now, but maybe these things need to be documented?  Or maybe I just missed where it was written?
